lib/models/batch_event.js

```diff
@@ -0,0 +1,33 @@
+export default class BatchEvent {
+    constructor(attributes) {
+        this.attributes = attributes
+    }
+
+    wrapAttributes = function () {
+        let event_object = {}
+
+        if (this.attributes.transactionId !== undefined && this.attributes.transactionId !== null)
```

Comment:
It might worth it to add a helper for this as it is repeated on many places

Reply:
Also, what if we want ti send a null value for the field?

Reply:
Good point - Let's plan to extract them into an helper on a separate PR.

Reply:
`transaction_id` is a mandatory field